core/server/guilds.js

```diff
@@ -47,6 +47,29 @@ const createGuild = params => {
   return guildId;
 };
 
+
+/**
+ * Create a new guild and emit event new_guild
+ * @fires member_changed
+ * @param {string} _guildId - The modified guild ID
+ *
+ * @returns {void}
+ */
+const emitMemberEvent = _guildId => {
```

Comment:
Wouldn't it be better to rename it to emitTeamMemberEvent?

Reply:
it's an internal and not exported function it's a little redondant no ? But it's ok